app/api/v1/models/election.py

```diff
@@ -46,6 +51,38 @@ class CiphertextElectionContext(Base):
     crypto_extended_base_hash: str
     """The `extended base hash code (𝑄')` in [ElectionGuard Spec](https://github.com/microsoft/electionguard/wiki)"""
 
+    @staticmethod
+    def stringToElementModP(value: str) -> ElementModP:
```

Comment:
To be flexible, it would be smart to allow both integers and hex strings to be passed in for ElementModP and ElementModQ . 
```py
from typing import Union

def to_element_mod_p(value: Union[int, str]) -> ElementModP
```

Reply:
I will do this, but it feels like YAGNI.  This method is in the CiphertextElectionContextDto and it is only ever called from within CiphertextElectionContextDto, and it will only ever pass strings because that's the declared data type of its properties: elgamal_public_key, commitment_hash, manifest_hash, crypto_base_hash, and crypto_extended_base_hash.

Reply:
I took at stab at the Union approach, but it increased the cyclomatic complexity which didn't feel quite right, so I added two new (unused, again feels like a violation of YAGNI) methods that accept ints in case anyone wants them in the future.  Please re-open this if you feel strongly about the Union approach and I'll go back to that.